src/settings/Plugins/PluginForm.js

```diff
@@ -31,21 +31,20 @@ class PluginForm extends React.Component {
   }
 
   renderPlugin(field, plugin) {
+    const intl = useIntl();
     const pluginType = this.props.pluginTypes[plugin.configName];
-    const options = [{
-      module: '@@',
-      displayName: '(none)',
-    }].concat(pluginType).map(p => ({
-      value: p.module,
-      label: `${p.displayName}${p.version ? ` v${p.version}` : ''}`,
-    }));
 
+    const options = [{ value: '@@', label: '(none)' }].concat(pluginType.map(p => ({
+      value: p.module,
+      label: intl.formatMessage({ id: 'ui-tenant-settings.settings.pluginNames.' + p.pluginType }) + ' ' + p.version,
```

Comment:
@K-Felk could you use string interpolation here?

Reply:
I don't see why not.